plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -504,12 +504,22 @@ function OPDSBrowser:genItemTableFromCatalog(catalog, item_url, username, passwo
                             title = link.title,
                         })
                     elseif link.rel == self.stream_rel then
+                        -- This for loop iterates through all keys in a
+                        --   Entry and looks for the count tag, then stores
+                        --   That key to use for updating the table value.
+                        local count_key = ""
+                        for k, v in pairs(link) do
+                            if string.find(k, ".:count") then
```

Comment:
This will only match single character namespaces?

Sidebar: Given the old code, this is a *trailing* match, so you can simplify this with something like `string.sub(k, -6, -1) == ":count"`.

Reply:
Indeed, this looks functionally identical to just `:count`.

Reply:
1. I See what you mean, I thought that the "." would be arbitrary length.
2. Another good idea, just look for the :count in the key rather than mess with pattern matching. I'll change it to this and see if it still works (Probably will, but you never know... ;) ).
3. You have to include the namespace as part of the count tag, per the OPDS-PSE Spec: https://anansi-project.github.io/docs/opds-pse/specs/v1.1

Specifically here:
```
<link rel="http://vaemendis.net/opds-pse/stream"
      type="image/jpeg"
      href="/opds-comics/stream/1217?page={pageNumber}&width={maxWidth}"
      pse:count="35"
      pse:lastRead="10"
/>
```
Where the count tag has the namespace, pse, in front of count. That whole <namespace>:count has to be the key, otherwise the page count is not pulled and defaults to 1, which only displays the cover.

Reply:
On the other hand, an anchored pattern would also work ;o). e.g., `string.find(blah, "^.+:count$")`.

One is probably less computationally intensive (string.sub) but at the cost of generating more garbage in Lua-land...

(I may not always have bright ideas when it's 6AM :D).

It's... unlikely to be of any real consequence given the amount of keys in the hash, though, so I'm fine with either approaches ;).